styles/styles.css

```diff
@@ -735,6 +768,38 @@ main .divider {
     border: 1px solid var(--neutral-sand);
 }
 
+main .development-time-section .columns>div>div {
+  margin-bottom: 32px
+}
+
+main .what-is-zelta-section .default-content-wrapper>p:last-child {
+  color: #d699ff;
+  margin-top: 48px;
+  letter-spacing: .01em;
+  line-height: 160%;
+  font-weight: 400;
+}
+
+main .section.what-is-zelta-section .default-content-wrapper>p {
+  margin: 24px 0;
+  font-size: 16px;
+  letter-spacing: .01em;
+  line-height: 160%;
+  font-weight: 300;
+}
+
+main .accelerate-section .default-content-wrapper:last-child p {
```

Comment:
Does this only relate to accelerate section? Or for all sections? Maybe this section can have a name of "no-margin" instead?